test/passing/let_binding_compact.ml

```diff
@@ -0,0 +1,21 @@
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+
+let f x = x + 1
```

Comment:
No it is not intended (but close enough at first). But it is fixed when doing what you suggested (instead of calling `is_simple` at all, just check `(width (sub_exp ~ctx x.pvb_expr)) <= c.margin`), I will revert the changes concerning `is_simple`.